compiler/src/iree/compiler/Dialect/Encoding/IR/EncodingAttrs.cpp

```diff
@@ -152,6 +152,45 @@ static int32_t getRoundedElementByteWidth(Type type) {
   return llvm::PowerOf2Ceil(byteAligned);
 }
 
+Value PackedStorageAttr::calculateStorageSizeInBytes(
+    Location loc, OpBuilder &builder, RankedTensorType type,
+    ValueRange dynamicDims) const {
+  unsigned elementBits = type.getElementTypeBitWidth();
+  assert(elementBits == 1 && "packed storage only supported for i1 for now");
+
+  // Calculate all static dims first, if any.
+  bool isPackedStorage = IREE::Encoding::hasPackedStorageAttr(type);
+  int64_t staticCount = 1;
+  if (!isPackedStorage) {
+    staticCount *= elementBits * 8;
```

Comment:
Let's avoid magic number: We can move `constexpr int kNumBitsInByte = 8` to global and use it in the file.

https://github.com/iree-org/iree/blob/27e7a9058eded6a20dc098368352329263f77a23/compiler/src/iree/compiler/Dialect/Encoding/IR/EncodingAttrs.cpp#L217